ai-missions/high-risk-flu-shot-mission/skills/update_member_phone_number/Readme.md

```diff
@@ -0,0 +1,35 @@
+### Update member phone number
+
+#### Description:
+For the given profile IDs will send requests to update phone number .
```

Comment:
Change to: For the Profile IDs derived from the configured Cohort, requests are sent to members to update their phone numbers.